service/src/main/kotlin/fi/espoo/evaka/messaging/MessageQueries.kt

```diff
@@ -417,6 +466,13 @@ group_accounts AS (
     JOIN daycare_group_placement dgp ON dgp.daycare_placement_id = p.id AND :today BETWEEN dgp.start_date AND dgp.end_date
```

Comment:
I think this should use `FROM placements p` instead of `relevant_placements`, because we're joining to daycare_group_placement and backup_care rows should not be used